tests/Http/Legacy/Web/TagsTest.php

```diff
@@ -81,6 +81,7 @@ public function testAddMultipleTagsAndDeleteOne()
         $post = factory(Post::class)->create();
         $post->tag('Good Submission');
         $post->tag('Tag To Delete');
+        $post = $post->fresh();
 
         // Make sure both tags actually exist
         $this->assertContains('Good Submission', $post->tagNames());
```

Comment:
yay I finally got the tests to pass! I'm not sure why but doing `$this->assertContains('Good Submission', $post->fresh()->tagNames());` didn't seem to work like it did below with `assertNotContains` so to keep things consistent, change to get a fresh version of the post always before any assertions.